charms/kfp-api/src/templates/auth_manifests.yaml.j2

```diff
@@ -1,109 +1,6 @@
 # Source manifests/apps/pipeline/upstream/base/installs/multi-user/api-service/cluster-role**.yaml
 # These manifest files have been modified to suit the needs of the charm; the app label, metadata name,
 # and namespace fields will be rendered with information from the application and the model.
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  labels:
-    app: {{ app_name }}
-  name: {{ app_name }}-role
-rules:
-- apiGroups:
-  - ''
-  resources:
-  - pods
-  - pods/log
-  verbs:
-  - get
-  - list
-  - delete
-- apiGroups:
-  - argoproj.io
-  resources:
-  - workflows
-  verbs:
-  - create
-  - get
-  - list
-  - watch
-  - update
-  - patch
-  - delete
-- apiGroups:
-  - kubeflow.org
-  resources:
-  - scheduledworkflows
-  verbs:
-  - create
-  - get
-  - list
-  - update
-  - patch
-  - delete
-- apiGroups:
-  - authorization.k8s.io
-  resources:
-  - subjectaccessreviews
-  verbs:
-  - create
-- apiGroups:
-  - authentication.k8s.io
-  resources:
-  - tokenreviews
-  verbs:
-  - create
-- apiGroups:
-  - tekton.dev
-  resources:
-  - pipelineruns
-  - taskruns
-  - conditions
-  - runs
-  - tasks
-  - customruns
-  verbs:
-  - create
-  - get
-  - list
-  - watch
-  - update
-  - patch
-  - delete
-- apiGroups:
-  - custom.tekton.dev
-  resources:
-  - pipelineloops
-  verbs:
-  - create
-  - get
-  - list
-  - watch
-  - update
-  - patch
-  - delete
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  labels:
-    app: {{ app_name }}
-  name: {{ app_name }}-binding
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: {{ app_name }}-role
-subjects:
-- kind: ServiceAccount
-  name: {{ app_name }}-sa
-  namespace: {{ namespace }}
----
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  labels:
-    app: {{ app_name }}
-  name: {{ app_name }}-sa
-  namespace: {{ namespace }}
```

Comment:
suggestion(blocking): I'm +1 to removing these but we should keep the PRs atomic, I suggest we create an issue describing why the roles are not needed and send a separate PR from this one removing them.